clang/docs/StandardCPlusPlusModules.rst

```diff
@@ -652,6 +652,134 @@ in the future. The expected roadmap for Reduced BMIs as of Clang 19.x is:
    comes, the term BMI will refer to the Reduced BMI and the Full BMI will only
    be meaningful to build systems which elect to support two-phase compilation.
 
+Experimental No Transitive Change
+---------------------------------
+
+Starting with clang19.x, we introduced an experimental feature: the non-transitive
```

Comment:
NIT: `Clang 19` ? I believe we start the name with an uppercase letter everywhere.

Reply:
Done